src/components/Upload/Upload.styles.ts

```diff
@@ -0,0 +1,85 @@
+import styled, { css } from 'styled-components';
+import color from '../../styles/colors';
+import resetCSS from '../../styles/reset';
+import { DivStyledProps } from './types';
+
+const DivStyled = styled.div<DivStyledProps>`
+    ${resetCSS};
+    align-items: center;
+    background: transparent;
```

Comment:
as color is set in Typography and bg color is set on upload complete i would think its best to set bg color here too, else if a user drops it onto a black or blue background dit will be a weird UX